doc/run/k8s/install-sqlflow.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     - containerPort: 8888
       name: notebook
     command:
-    - bash
```

Comment:
Thanks for fixing this bug.
I believed Jupyterhub spawn Jupyter notebook server using the script: https://github.com/sql-machine-learning/sqlflow/blob/develop/docker/jupyterhub/jupyterhub_config.py#L69